src/phenopacket_mapper/data_standards/code_system.py

```diff
@@ -49,67 +49,128 @@ def __contains__(self, item):
             return self == item.system
 
 
+NCBITaxon = CodeSystem(
+    name='NCBI organismal classification', 
+    namespace_prefix='NCBITaxon', 
+    url='https://www.ncbi.nlm.nih.gov/taxonomy'
+)
+
+GENO = CodeSystem(
+    name='GENO: The Genotype Ontology', 
+    namespace_prefix='GENO', 
+    url='http://www.genoontology.org/'
+)
+
+SO = CodeSystem(
+    name='Sequence types and features ontology', 
+    namespace_prefix='SO',
+    url='http://www.sequenceontology.org/'
+)
+
+ICD9 = CodeSystem(
+    name='International Classification of Diseases, Ninth Revision',
+    namespace_prefix='ICD9',
+    url='https://www.cdc.gov/nchs/icd/icd9.htm',
+    synonyms=['ICD-9', 'ICD_9', 'icd9']
```

Comment:
```suggestion
    synonyms=['ICD-9', 'ICD_9']
```

case is ignored when matching